autoload/neomake/makers/ft/apiblueprint.vim

```diff
@@ -0,0 +1,16 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#apiblueprint#EnabledMakers() abort
+    return executable('drafter') ? ['drafter'] : []
+endfunction
+
+function! neomake#makers#ft#apiblueprint#drafter() abort
+    " Drafter only operates on a single file at a time, and therefore doesn't
+    " bother to print out a file name with each error. We need to attach this
+    " so that the quickfix list can function properly
+    return {
+        \ 'args': ['-l', '-u'],
+        \ 'errorformat': '%f: %t%[%^:]\\+: (%n) %m; line %l\, column %c%.%#',
+        \ 'mapexpr': 'expand("%:p") . ": " . v:val'
```

Comment:
I think this should use `mapexpr_setup_bufvars` from https://github.com/neomake/neomake/pull/630 (not yet merged).
I will look into adding tests for it, and get it in - but maybe only during the next days.